RecoPixelVertexing/Configuration/python/RecoPixelVertexing_cff.py

```diff
@@ -4,7 +4,22 @@
 #
 # for STARTUP ONLY use try and use Offline 3D PV from pixelTracks, with adaptive vertex
 #
-#from RecoPixelVertexing.PixelVertexFinding.PixelVertexes_cff import *
-from RecoVertex.PrimaryVertexProducer.OfflinePixel3DPrimaryVertices_cfi import *
+from RecoPixelVertexing.PixelVertexFinding.PixelVertexes_cff import *
+#from RecoVertex.PrimaryVertexProducer.OfflinePixel3DPrimaryVertices_cfi import *
```

Comment:
Mhm, this is probably not intended / wanted upstream.

Reply:
@slava77 @jpata @mtosi @vmariani can you let me know if this change is OK ?

Reply:
- PixelVertexes_cff leads to a definition of `pixelVertices = cms.EDProducer("PixelVertexProducer",` with `DivisiveVertexFinder`
- OfflinePixel3DPrimaryVertices_cfi  defines `pixelVertices = cms.EDProducer("PrimaryVertexProducer"`  with `DA_vect` algorithm

The switch to the latter happened in 2009 https://github.com/cms-sw/cmssw/commit/d61d086b5012b01904798f3bcfb0b2e933194261
Currently, the `pixelVertices` are not running in anything offline that's used for analysis/production.

IIUC, `pixelVertexFromSoA` is more representative of `DivisiveVertexFinder` (only Z is computed, x,y is set by the beamline).

So, for the purpose of comparing CPU and GPU variants, the update is perhaps OK.

It would be nice to have a confirmation from the TRK POG @mtosi @vmariani